### PR TITLE
OKTA-641384: clear token from cache on call to oauth.clear_access_token()

### PR DESCRIPTION
**Pull request by bryanapellanes-okta**
Clear access token from cache on call to OAuth.clear_access_token().

### CODE REVIEW

File: okta/api_response.py [hunk 1]
```diff
@@ -182,5 +182,6 @@ async def get_next(self):
                 next_response = OktaAPIResponse(
                     self._request_executor, req, res_details, resp_body)
                 self._next = next_response._next
+                self._resp_headers = res_details.headers
```
Comment (justinabrokwah-okta) [inline okta/api_response.py:185]: Is `self._resp_headers` used anywhere?

Comment (bryanapellanes-okta) [reply okta/api_response.py:185]: That's a good question.  @haggrip are you able to provide further insight here?

**Review by justinabrokwah-okta — APPROVED** [review]
✅

**Review by duytiennguyen-okta — APPROVED** [review]

**Review by arvindkrishnakumar-okta — APPROVED** [review]

**Comment by bryanapellanes-okta** [comment]
@haggrip I'm excited to merge your contribution, before I do so we have some formalities to address.  Please review the "Okta Individual Contributor License Agreement" here: https://developer.okta.com/cla/.  TL;DR For legal reasons, mark the PR an "Obvious Fix" or sign a CLA.

**Comment by haggrip** [comment]
CLA was signed

**Pull request merged**